File: Iniciante/1020 - Idade em Dias/C#/1020 - Idade em Dias.cs
using System;

class URI {
    static void Main() {
        int idade = Int32.Parse(Console.ReadLine());

        int anos = idade/365;
        int meses = (idade%365)/30; // resto de anos dividido por meses
        int dias = (idade%365)%30; // resto de meses 

        Console.WriteLine("{0} ano(s)\n{1} mes(es)\n{2} dia(s)", anos, meses, dias);
    }
}

File: Iniciante/1020 - Idade em Dias/Python/1020 - Idade em Dias.py
# entrada da idade em dias
idade = int(input())

# operações matemáticas
anos = int(idade/365) # anos = inteiro de idade em dias/365
meses = int((idade%365)/30) # meses = inteiro do resto de anos/30
dias = int((idade%365)%30) # dias = inteiro do resto de meses

# saída da idade formatada
print(anos, 'ano(s)')
print(meses, 'mes(es)')
print(dias, 'dia(s)')